### PR TITLE
Api optimisation: Audio processing api seperation done. 

### PR DESCRIPTION
**Pull request by DevendraPPatil**
-

### CODE REVIEW

File: routes.py [hunk 1]
```diff
@@ -67,8 +63,18 @@ async def audio_processing(data: audioData):
             # Delete audio data from cache
             del audio_data
             del audio_io
-
-            return {"denoised_audio_base64": denoised_audio_base64}
+ 
+            return {
+                "denoised_audio_base64": denoised_audio_base64,
+                "insertion": error_arrays['insertion'],
```
Comment (sudeeppr1998) [inline routes.py:69]: Why this text eval matrices added here in audio processing function ?

File: routes.py [hunk 2]
```diff
@@ -53,12 +52,17 @@ async def get_phonemes(data: PhonemesRequest):
...
-        audio_base64_string = data.audio_base64
+
+    base64_string = data.base64_string
+    
+    if data.base64_string:
+        audio_base64_string = data.base64_string
         if audio_base64_string:
             # Convert base64 audio to audio data
             audio_data = base64.b64decode(audio_base64_string)
             audio_io = BytesIO(audio_data)
```
Comment (sudeeppr1998) [inline routes.py:63]: you have done BytesIO conversion here. same you are doing in get_pause_count function. please check

File: utils.py [hunk 3]
```diff
@@ -111,18 +111,16 @@ def convert_to_base64(audio_data, sample_rate):
...
+    insertion_chars = []
+    deletion_chars = []
     substitution = []
-
+    result = []
     for chunk in alignments[0]:
         if chunk.type == 'insert':
-            insertion.extend(
-                list(range(chunk.hyp_start_idx, chunk.hyp_end_idx)))
+            insertion_chars.update(range(chunk.hyp_start_idx, chunk.hyp_end_idx))
```
Comment (sudeeppr1998) [inline utils.py:121]: @DevendraPPatil 
If this code change is related to audio pause count ??

**Comment by sureshemis** [comment]
many are outdated

**Pull request closed without merging**